src/test/scala/nl.knaw.dans.easy.solr4files/UpdateServletSpec.scala

```diff
@@ -145,7 +145,7 @@ class UpdateServletSpec extends TestSupportFixture
 
   it should "complain about the required query" in {
     delete("/") {
-      body shouldBe "delete requires param 'q': a solr query"
+      body shouldBe "delete requires param 'q': got []"
```

Comment:
Is this the actual message that users get to see? I don't like the `[]`. You better return something useful here!